#Destination Tracking

#True: Destination Present in Frame #False: Destination absent
bool destPresent

#Destination rectangle is given by (x,y) and (x+w, y+h)
int32 destX
int32 destY
int32 destWidth
int32 destHeight
